README.md

```diff
@@ -9,27 +9,82 @@ Note that at this time the Package Manager has no support for iOS, watchOS, or t
 ---
 
 ## Table of Contents
-
-* [Contributions](#contributions)
+* [Getting Started](#getting-started)
+* [Documentation](#documentation)
 * [System Requirements](#system-requirements)
 * [Installation](#installation)
   * [Managing Swift Environments](#managing-swift-environments)
   * [Choosing a Swift Version](#choosing-a-swift-version)
-* [Documentation](#documentation)
 * [Support](#support)
+* [Contributions](#contributions)
 * [License](#license)
 
 ---
 
-## Contributions
+## Getting Started
+🧐 Verify Installation 
```

Comment:
Lets avoid using emojis for now. I am a big fan of them but it is kind of weird to see them in docs.

Reply:
😅sure